airbyte-integrations/connectors/destination-iceberg-v2/src/main/kotlin/io/airbyte/integrations/destination/iceberg/v2/io/IcebergUtil.kt

```diff
@@ -162,27 +179,36 @@ class IcebergUtil {
      * @return The Iceberg [Catalog] configuration properties.
      */
     fun toCatalogProperties(icebergConfiguration: IcebergV2Configuration): Map<String, String> {
+        // Nessie does not allow explicitly setting the S3 bucket region.
+        // Instead, it relies on a static credentials provider, which retrieves the region
+        // from the system property. Therefore, we need to set the region this way.
+        System.setProperty(
```

Comment:
for (non-docker) integration tests: I guess this is fine, since we can just use the same aws region for all test cases?